composer.json

```diff
@@ -7,8 +7,12 @@
     }
   },
   "require": {
-    "php": ">=5.5",
-    "symfony/symfony": "^2.8 || ^3.0 || ^4.0",
```

Comment:
previous version should be supported, eg. `^4.0 || ^5.0`

Reply:
added